src/Workspaces/Core/Portable/Execution/PinnedRemotableDataScope.cs

```diff
@@ -33,6 +33,11 @@ internal sealed class PinnedRemotableDataScope : IDisposable
             _storages.RegisterSnapshot(this, storage);
         }
 
+        /// <summary>
+        /// This indicates whether this scope is for primary branch or not (not forked solution)
+        /// </summary>
```

Comment:
it tells whether the solution this checksum is created from is from primary branch or not. it depends whether the consumer care about the solution is from primary branch or not. for example, OOP framework care.